### ISSUE TITLE
Asking for improvements in browsing mode in outlook

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#7794]
### Steps to reproduce:
open the attached e-mail.

navigate with the arrows below
see that nvda will read as follows.
Este email não está sendo exibido corretamente? Ver no seu navegador . 
Se deseja se desinscrever, clique aqui por favor.
now follow 6 empty lines, and then the text
no FineReader 14
now press up arrow
see that the text is read
50% de desconto 

The problem is that nvda does not read the text
50% de desconto 

when navigating with the arrows below, only when you return with the arrow above.

Another question, navigate with the arrows below until you hear the text:

P.O. Box #20, Moscou, Rússia, 127273 | Tel: +7 (495) 783 3700 | Fax: +7 (495) 783 2663

now follows two problems:
press up arrow and see that the text is read
(C) 2017 ABBYY 3A
which was not read when walking with arrows below, similar to the first problem reported.
Now navigate with arrows above and see that it is always read empty line, and you can not return to the beginning of the email with the above arrows, you can only navigate to the beginning if you press the left arrow or ctrl + home arrow.
to read the text correctly press ctrl + a and read the selection with nvda + shift + s.

NVDA version:
> Example: next-14660

NVDA Installed or portable:
instaled
Other information:
> Example: Running in a VM

Windows version:
> Example: Windows 10 Version 1709 Build 16299.64

Name and version of other software in use when reproducing the issue:
outlook 2016 16.0.8625.2132

### Other questions:

Does the issue still occur after restarting your PC?
yes

Have you tried any other versions of NVDA?
> Please list them and the result

yes

The same results.
This problem occurs with similarity with other emails mainly received from social networks and store promotions.
Nvda is in browse mode.

see that many empty lines are read, it would be interesting if there were not so many empty lines when navigating with the arrows, but the priority in my opinion is to solve the problems reported above.

[Black Friday está aqui.zip](https://github.com/nvaccess/nvda/files/1501750/Black.Friday.esta.aqui.zip)

**Comment by Adriani90** [comment]
I can reproduce this with NVDA 2017.2 and NVDA 2017.3 in MS Outlook 2010 on Windows 8.1.

**Comment by Adriani90** [comment]
Moreover, if you activate focus mode it is not possible to read the e-mail properly at all. NVDA gets stuck on the third line or so.

**Comment by Adriani90** [comment]
However, if you hold pressed ctrl and navigate by paragraphs with up and down arrow or by words with left and right arrow all text is recognized properly and you can read the e-mail without problems in browse and focus mode.

**Comment by fernando-jose-silva** [comment]
Thank you very much for testing.

**Comment by feerrenrut** [comment]
Loading the email in Chrome allows it to be read successfully as well. I think there could be improvement to way that emails with HTML content work in outlook, there seems to be something strange going on. In the meantime, as a workaround you can view these messages in a browser and read them with NVDA there.

**Comment by TommasoNonis90** [comment]
Hello all,
I would like to bring your attention to this issue as this is really a deal braker for me, as I have to frequently work with html emails that contain  multiple layers of layout tables, often nested into each other. These tables are only for responsiveness purposes, and have no data whatsoever. They are used by newsletter template writers to make the content auto-scale when it is viewed on a mobile device vs a desktop.
I recall this problem was already brought to Mick's and JAmie's attention, but they came to the conclusion that virtualization wouldn't be necessary, as there's no virtualization happening in Word anyway.
But here, Word is used to render what would be html code, and viewing it the way word displays it, even with browse mode active, is a real pain with  a screen reader.
This is exactly why the competition has a setting to virtualize outlook emails in the buffer for ease of reading.

**Comment by TommasoNonis90** [comment]
I'd also like to point out that, if UI automation is disengaged, the layout tables are not actually spoken, but all cells are navigated when pressing down arrow, resulting in many many blank lines. With UI automation on for word, all hese html layout tables (often with only one row and one column) are announced.

**Comment by seanbudd** [comment]
@fernando-jose-silva - is this issue still relevant?

**Comment by fernando-jose-silva** [comment]
Thank you very much, I no longer noticed this problem.
